comp/dogstatsd/server/intern.go

```diff
@@ -78,7 +78,11 @@ func (i *stringInterner) LoadOrStore(key []byte) string {
 			i.curBytes = 0
 		}
 
-		i.strings = make(map[string]string)
+		// Clear strings of keys, leave the underlying allocation
+		// intact.
+		for k := range i.strings {
+			delete(i.strings, k)
+		}
```

Comment:
Is this different from `clear(i.strings)`?

Reply:
Ah, introduced in 1.21. Neat. I think the use of the built-in makes more sense.

Reply:
Ah, can't use 1.21 yet.